website/content/docs/platform/k8s/vso/openshift.mdx

```diff
@@ -52,11 +52,11 @@ controller:
   kubeRbacProxy:
     image:
       repository: registry.redhat.io/openshift4/ose-kube-rbac-proxy
-      tag: v4.13.0
+      tag: v4.14.0
```

Comment:
We may want to wait on updating this until we also update the default version in the OLM[ bundle kustomize](https://github.com/hashicorp/vault-secrets-operator/blob/v0.4.0/config/default-openshift/manager_ubi_auth_proxy_patch.yaml#L17), probably in the next release.

Reply:
Reverted in 8b18f29d0be88ae2ee47381c6a651156e22dcc6e - sigh